certified-connectors/Fliplet/apiDefinition.swagger.json

```diff
@@ -36,36 +36,6 @@
         }
       }
     },
-    "/v1/apps/{appId}": {
```

Comment:
@squallstar The operation has been deleted. This change breaks existing flows. can you please confirm that no existing flows or users will be affected by this change?

Reply:
@Amjed-Ayoub this operation has been deleted because it doesn’t provide the required value to the users while introducing some error risks. It isn’t used in any flow and won’t break anything for users.

Reply:
@squallstar thank you so much for the confirmation.